lib/rubocop/cop/rails/time_zone.rb

```diff
@@ -44,9 +44,9 @@ class TimeZone < Cop
                             :to_i, :to_f]
 
         def on_const(node)
-          _module, klass = *node
+          mod, klass = *node
```

Comment:
Of course. Silly me.

On Sunday, June 28, 2015, Vladimir Dementyev <notifications@github.com>
wrote:

> In lib/rubocop/cop/rails/time_zone.rb
> <https://github.com/bbatsov/rubocop/pull/1997#discussion_r33421613>:
>
> > @@ -44,9 +44,9 @@ class TimeZone < Cop
> >                              :to_i, :to_f]
> >
> >          def on_const(node)
> > -          _module, klass = *node
> > +          mod, klass = *node
>
> module is reserved keyword
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bbatsov/rubocop/pull/1997/files#r33421613>.
>


-- 
Best Regards,
Bozhidar Batsov

http://www.batsov.com